includes/wc-core-functions.php

```diff
@@ -389,6 +390,7 @@ function get_woocommerce_currency_symbol( $currency = '' ) {
 		'TRY' => '&#8378;',
 		'TWD' => '&#78;&#84;&#36;',
 		'UAH' => '&#8372;',
+		'UGX' => 'UGX',
```

Comment:
https://www.facebook.com/dwampamba/posts/1595968117394934?comment_id=1596231997368546&notif_t=feed_comment&notif_id=1459711042081842
Link of a survey made on facebook, is proof UGX is widely used than the USh.